app/xds_api/xapi.py

```diff
@@ -3,14 +3,12 @@
 
 
 VERB_WHITELIST = {
-    "https://w3id.org/xapi/tla/verbs/socialized",
     "https://w3id.org/xapi/tla/verbs/explored",
-    # TODO: there should only be one explored verb
-    "https://w3id.org/xapi/acrossx/verbs/explored",
-    "https://w3id.org/xapi/acrossx/verbs/prioritized",
-    "https://w3id.org/xapi/dod-isd/verbs/curated",
-    "https://w3id.org/xapi/tla/verbs/registered",
     "https://w3id.org/xapi/acrossx/verbs/searched",
+    "http://activitystrea.ms/save",
+    "https://xapi.edlm/profiles/edlm-ecc/concepts/verbs/curated",
+    "http://adlnet.gov/expapi/verbs/shared",
```

Comment:
Can these be `https` instead of `http`?  Also the `viewed` from tincanapi seems to be the only one currently accessible at that address.  I assume the others are resolvable using these IRIs, but should they be accessible at that address?

Reply:
In general we try to stay to the canonical form of a given IRI (as listed on the ADL profile server or used in active systems), even if that is HTTP. Resolvability is preferred where possible but isn't a hard requirement. You'll note that in the case of IRIs from our profiles like `https://xapi.edlm/profiles/edlm-ecc/concepts/verbs/curated` the domain is very much contrived. All that said http without the s bothers me on a ideological level so I'll check on those and see if there is precedent for using the https versions.

Reply:
Checked up on those, and unfortunately this screenshot illustrates the issue: 
<img width="1016" alt="image" src="https://github.com/user-attachments/assets/cab4e688-1c16-40a5-886f-3badec753e16" />